src/app/DeviceProxy.h

```diff
@@ -57,18 +55,14 @@ class DLL_EXPORT DeviceProxy
 
     virtual chip::Optional<SessionHandle> GetSecureSession() const = 0;
 
-    virtual bool IsActive() const { return true; }
-
     virtual CHIP_ERROR SetPeerId(ByteSpan rcac, ByteSpan noc) { return CHIP_ERROR_NOT_IMPLEMENTED; }
 
-    const ReliableMessageProtocolConfig & GetMRPConfig() const { return mMRPConfig; }
+    const ReliableMessageProtocolConfig & GetRemoteMRPConfig() const { return mRemoteMRPConfig; }
 
 protected:
     virtual bool IsSecureConnected() const = 0;
 
-    virtual uint8_t GetNextSequenceNumber() = 0;
-
-    ReliableMessageProtocolConfig mMRPConfig = GetLocalMRPConfig();
+    ReliableMessageProtocolConfig mRemoteMRPConfig = GetLocalMRPConfig();
```

Comment:
With the renaming, `GetLocalMRPConfig` seems like an odd thing to init this to: it makes _our_ SED state affect the _remote_ settings.  Followup to address this, please?

Reply:
We should renmae `GetLocalMRPConfig` to `GetDefaultMRPConfig`, this is not the only place it is assigned to remote MRP config

Reply:
#17620